Core/Newsletter2Go/Components/Api/Resource/NewsletterCustomer.php

```diff
@@ -2,10 +2,8 @@
 
 namespace Shopware\Components\Api\Resource;
 
-use Shopware\Models\CustomerStream\Mapping;
 use Shopware\Models\Newsletter\Address;
 use Doctrine\ORM\Query\Expr;
-use Shopware\Models\Newsletter\Group;
 use Shopware\Models\Plugin\Plugin;
 
```

Comment:
Can you explain to me why we can remove CustomerStream Mapping and Newsletter Group here? :)

Reply:
I removed them because they were not used using those namespaces.
So the purpose here is just to clean up, if the are not needed